prov/efa/src/efa_cq.h

```diff
@@ -305,6 +305,11 @@ static inline int efa_write_error_msg(struct efa_base_ep *ep, fi_addr_t addr,
 	return 0;
 }
 
+static inline uint32_t efa_cq_wc_read_qp_num(struct efa_ibv_cq *cq)
+{
+	return ibv_wc_read_qp_num(cq->ibv_cq_ex);
```

Comment:
Can you drop this inline function and use ibv directly? That will make my life easier because the PR https://github.com/ofiwg/libfabric/pull/11217 is introducing wrapper that does the same thing